deploy/stacks/cloudfront.py

```diff
@@ -365,7 +353,7 @@ def __init__(
                     route53_targets.CloudFrontTarget(cloudfront_distribution)
                 ),
             )
-        if userguide_alternate_domain:
+        if userguide_alternate_domain and custom_auth != None:
```

Comment:
very very minor: let's be consistent and use `is not None` as we have done when evalutaing the variable in other places